esmvalcore/preprocessor/_area.py

```diff
@@ -520,8 +573,15 @@ def extract_shape(cube, shapefile, method='contains', crop=True,
             pad_hawaii = True
 
         if crop:
+            lon_min, lat_min, lon_max, lat_max = _get_bounds(
```

Comment:
lon and lat seem to flipped compared with line 457. Which is correct? The other one should be changed. And if possible let's make sure that the tests would catch this, i.e. choose a region for tests with longitudes outside [-90., 90.].

Reply:
Good find, I will have a look on Friday. I always mix those around 😅

Reply:
Alright, I'll bump it to 2.4.0 then.

Reply:
Should be fixed now 👍

The code was working correctly, I just mixed up the variable names in the function. The tests would catch this if the coordinates were be flipped (I just checked).